tests/functional/pv/pv_services/test_cephfs_pvc_with_chunk_size.py

```diff
@@ -0,0 +1,97 @@
+import time
+import pytest
+import logging
+
+from ocs_ci.framework.testlib import bugzilla, tier2
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import pod
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import cluster
+from ocs_ci.framework.pytest_customization.marks import (
+    green_squad,
+    skipif_external_mode,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@bugzilla("2171225")
+@pytest.mark.polarion_id("OCS-5772")
+@green_squad
+@skipif_external_mode
+class TestCephfsWithChunkIo:
+    """
+    This class takes care of create Cephfs PVC, create Fedora dc pod and run Chunk IO on fedora pod
+    """
+
+    def test_cephfs_with_large_chunk_io(self, pvc_factory, dc_pod_factory):
+
+        """
+        This function facilitates
+        1. Create PVC with Cephfs
+        2. Create dc pod with Fedora image
+        3. Copy helper_scripts/chunk.py to Fedora dc pod
+        4. Set debug 25 for mds in rook-ceph-tools pod
+        5. Run chunk.py on fedora pod for 15mins
+        6. Read mds pod logs and look for errors
+        7. If no errors seen in the mds.log then the test will Pass.
+        8. If any errors found then the test will fail.
+        9. Check for warning "MDS_CLIENT_LATE_RELEASE" in ceph.
+        10. If warning found, test will fail.
+        """
+        file = constants.CHUNK
+        interface = constants.CEPHFILESYSTEM
+
+        # Creating PVC with cephfs as inetrface
+        log.info(f"Creating {interface} based PVC")
+        pvc_obj = pvc_factory(interface=interface, size="50")
+        # Creating a Fedora dc pod
+        log.info("Creating fedora dc pod")
+        pod_obj = dc_pod_factory(pvc=pvc_obj, interface=interface)
+        # Copy chunk.py to fedora pod
+        log.info("Copying chunk.py to fedora pod ")
+        cmd = f"oc cp {file} {pvc_obj.namespace}/{pod_obj.name}:/"
+        helpers.run_cmd(cmd=cmd)
```

Comment:
since you have the pod_obj,  you could use copy_to_pod_rsync func to copy the file from a local path to the pod path